driver-core/src/main/com/mongodb/internal/async/client/AsyncListDatabasesIterable.java

```diff
@@ -67,4 +67,15 @@
      * @mongodb.server.release 3.4.3
      */
     AsyncListDatabasesIterable<T> nameOnly(@Nullable Boolean nameOnly);
+
+    /**
+     * Sets the authorizedDatabasesOnly flag that indicates whether the command should return just the databases which the user
+     * is authorized to see.
+     *
+     * @param authorizedDatabasesOnly the authorizedDatabasesOnly flag, which may be null
+     * @return this
+     * @since 4.1
+     * @mongodb.server.release 4.0.5
```

Comment:
I guess it was backported?  Still, just call it 4.0, as we don't generally refer to patch releases here